test/unit/assertions.test.js

```diff
@@ -7,10 +7,10 @@ const pathToApiSpec = path.resolve('test/data/exampleOpenapi.yml');
 chai.use(chaiResponseValidator(pathToApiSpec));
 const { expect } = chai;
 
-describe('expect(res)', function () {
+describe('expect(res).to', function () {
   describe('satisfyApiSpecForStatus(expectedResStatus)', function () {
-    describe('matching API spec value', function () {
-      describe('when the response matches the api spec', function () {
+    describe('when the API spec defines the expected response', function () {
+      describe('and when the spec matches the response', function () {
```

Comment:
I don't think using `and` in our describe blocks makes them clearer, I propose we reduce the words around these describe blocks to match up with the other tests:
```suggestion
      describe('spec matches the response', function () {
```